### PR TITLE
Create appsmith_templates.md

### PR DESCRIPTION
**Pull request by akshayrangasaid**
> Pull Request Template
>
> Use this template to quickly create a well written pull request. Delete all quotes before creating the pull request.

Fixes # (issue)
> if no issue exists, please create an issue and ask the maintainers about this first

## Type of change

> Please delete options that are not relevant.

- Bug fix (non-breaking change which fixes an issue)
- New feature (non-breaking change which adds functionality)
- Breaking change (fix or feature that would cause existing functionality to not work as expected)
- This change requires a documentation update

## How Has This Been Tested?

> Please describe the tests that you ran to verify your changes. Provide instructions, so we can reproduce. 
> Please also list any relevant details for your test configuration.

- Test A
- Test B

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: appsmith_templates.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+## Appsmith Example Applications and Templates
+
+Our team and the community have created a few **example** applications that can be used as is, or as a template which can serve as a starting point for your application.
+
+You can use the applications for yourself following the simple steps below:
+
+1. Log in to Appsmith
```
Comment (Nikhil-Nandagopal) [inline appsmith_templates.md:7]: Login can happen after fork right?
Can't the user first browse then fork then login?

File: appsmith_templates.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+1. [Applicant Tracker System](https://app.appsmith.com/applications/5f43bce13697fd5df1c52ce4/pages/5f43c39c3697fd5df1c52d50)
+
+> This application will need you to connect to your instance of Postgres, but once done, you will be able to manage the entire lifecycle of applicants in your hiring pipeline. We've also added Google Calendar API calls so you can schedule interviews directly from this application.
+
+2. [Calendly in Appsmith](https://app.appsmith.com/applications/600186605b452f525458d6d2/pages/600186605b452f525458d6d4)
+
+
+> A barebones version of calendly that you can use to schedule meetings, or just play around with how you can use APIs and Appsmith to build really powerful applications. Connects to Google Calendar API, so will need you to [authorize your OAuth2 token](https://docs.appsmith.com/core-concepts/connecting-to-data-sources/connect-to-apis/authentication/oauth2-authentication).
+
+3. [Founder/Investor CRM](https://app.appsmith.com/applications/6098bdc65864501cc39c3d2f/pages/6098bdc65864501cc39c3d31)
```
Comment (Nikhil-Nandagopal) [inline appsmith_templates.md:27]: What about the Gmail support app and the mongo admin panel?

File: appsmith_templates.md (hunk 1 above)
Comment (akshayrangasaid) [reply appsmith_templates.md:7]: Yep they can, this is just a suggested flow, but forking will in the end need login so put it upfront,.

File: appsmith_templates.md (hunk 2 above)
Comment (akshayrangasaid) [reply appsmith_templates.md:27]: Will prettify and add them tomorrow.

File: appsmith_templates.md (hunk 1 above)
Comment (akshayrangasaid) [reply appsmith_templates.md]: Changed it

**Review by Nikhil-Nandagopal — DISMISSED** [review]

**Comment by mohanarpit** [comment]
@akshayrangasaid Is this PR still valid? If yes, please merge it in.

**Pull request merged**